### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Web Server-Certificates: When no records are found after searching with any keyword in search box, there is no on-screen or programmatically determined instructions available for the users to know about the search results.

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#137]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Certificates and press enter
6) Verify any on screen instructions are provided for the users on searching with any keyword in search box

**Actual Result:**
There is a blank page observed on searching with any keyword in search box , no on-screen or programmatically determined instructions available for the users to know whether the search results exists or not. 

**Expected Result:**
Some message or information should be provided to the user. They should get proper instructions of records not being found after searching with any keyword or word in the search box 

**User Impact:** 
The No records found message is useful for displaying instances when a query does not return any records. If the user wants to conduct a quick search with any keyword inside a  grid for a field in the table but that keyword does not exist in the table, for that this type of message is helpful for the user.  

**Attachments:**  
[Bug_137_There is no on-screen message shown to the user after searching with any keyword.pptx](https://github.com/Microsoft/IIS.WebManager/files/2327604/Bug_137_There.is.no.on-screen.message.shown.to.the.user.after.searching.with.any.keyword.pptx)

**Comment by jhkimnew** [comment]
Fixed.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yUsable; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Edge; #C+E-H; #.NETWebsites; #Chrome
